packages/venia-concept/src/components/MiniCart/emptyMiniCart.css

```diff
@@ -16,9 +16,10 @@
 .continue {
     composes: root from '../Button/button.css';
     color: white;
-    background-color: rgb(var(--venia-teal));
+    background-color: rgb(var(--color));
 }
 
 .continue:hover {
-    color: rgb(var(--venia-teal));
+    color: white;
+    background-color: rgb(var(--venia-teal));
```

Comment:
@soumya-ashok what is the desired state for hover? [The mocks](https://magento.invisionapp.com/share/WQN5F7BYBPG#/screens/316856656) show the black background but I don't think inVision lets you do hover states in mocks (or I just cant find one).

Reply:
@sirugh - The general pattern Jimmy and I decided on is that the button is dark gray in its static state and turns to teal in the hover or focused state.